WayneSociety_Project/static/css/Services.css

```diff
@@ -1,12 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
 h5 {
     font-size:x-large;
 }
 
-.card-deck{
-  font:white;
-  opacity:0.75; /* makes the page transparent*/
-  /*used so the cards will be side to side*/
-  display: inline-block;
-  /*width: 669px;*/
 
+/*float 3 columns side by side*/
+.column {
```

Comment:
Good Use of comments on your code, will make things easy for the rest of us to debug when there is an issue